frontend/src/features/contacts/contact/create/CreateContactContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import { ContactBreadcrumb, ContactTypes, ContactTypeSelector } from 'features/contacts';
+import * as Styled from 'features/contacts/styles';
+import { getIn } from 'formik';
+import React from 'react';
+import { useHistory, useLocation, useRouteMatch } from 'react-router-dom';
+import styled from 'styled-components';
+
+import ContactRouter from './ContactRouter';
+
+export const CreateContactContainer: React.FunctionComponent = () => {
+  const history = useHistory();
+  const { path } = useRouteMatch();
+  const { pathname } = useLocation();
+
+  const contactType = getContactTypeFromPath(pathname);
+
+  const onSelectorChange = (newValue: ContactTypes) => {
+    history.push(`${path}/${newValue}`);
+  };
+
+  return (
+    <ContactLayout>
+      <ContactBreadcrumb />
+      <Styled.H1>Add a Contact</Styled.H1>
+
+      <ContactTypeSelector
+        contactType={contactType}
+        setContactType={onSelectorChange}
+      ></ContactTypeSelector>
+
+      <ContactRouter />
+    </ContactLayout>
+  );
+};
+
+const ContactLayout = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  height: 100%;
+  width: 50%;
+  min-width: 30rem;
+  overflow: hidden;
+  padding: 1rem;
+  gap: 1.6rem;
+`;
```

Comment:
Since it is already importing ```import * as Styled from 'features/contacts/styles';``` could this be moved there?

Reply:
we could but would need a different naming. `contacts/styles` covers all common styling (for contacts list, create forms, update forms in the future etc). "ContactLayout" is too generic